arch-makepkg.md

```diff
@@ -0,0 +1,50 @@
+# Installing SystemVerilog on Arch-based systems using Pacman and makepkg
+
+In this guide we'll show you how to install all necessary tools for synthesizing or simulating your SystemVerilog projects.
+
+> "[makepkg](https://git.archlinux.org/pacman.git/tree/scripts/makepkg.sh.in) is a script to automate the
```

Comment:
In order to use `makepkg` on a newly installed Manjaro system, I've had to install the dependencies `binutils` and `fakeroot`. It would be nice, if you could mention those.